gubernator/templates/pr_dashboard.html

```diff
@@ -63,7 +63,7 @@ <h3 class="pr-section">
 % endfor
 
 <div id="help">
-<p><b>Needs Attention</b> is a heuristic grouping based on a <a href="https://github.com/kubernetes/test-infra/blob/master/gubernator/github/classifier.py#L288">simple state machine</a>. Use the "ACK" button to remove a PR from "Needs Attention" until something new occurs.
+  <p><b>Needs Attention</b> is a heuristic grouping based on a <a href="https://github.com/kubernetes/test-infra/blob/master/gubernator/github/classifier.py#L288">simple state machine</a>. <a href="/pr">Login</a> to use the "ACK" button to remove a PR from "Needs Attention" until something new occurs.
```

Comment:
Make the login prompt only show if the user isn't logged in:

    {% if user != login or not login %}<a href="/pr">Login</a> to use{% else %}Use{% endif %}